metricbeat/helper/metricset.go

```diff
@@ -52,12 +52,19 @@ func (m *MetricSet) Fetch() error {
 		go func(h string) {
 			defer m.Module.wg.Done()
 
+			starttime := time.Now()
 			event, err := m.MetricSeter.Fetch(m, h)
+			endtime := time.Now()
```

Comment:
Adjusted to the way you suggested.

Good point about the precision clock. Wasn't thinking of that but there doesn't see to be a better solution currently?